packages/zod-form-data/README.md

```diff
@@ -190,26 +190,20 @@ const schema = zfd.formData({
 ```
 
 There is a unique case in Remix when using a CustomUploadHandler, 
-the field will be a `File` on the client side, but an ID string (or URL) after uploading on the server.
+the field will be a `File` on the client side, but an ID string / number / url after uploading on the server.
 
-In this case you will need the schema to switch to string on the server:
+In this case you will need the schema to switch types on the server, and that's where `zfd.fileObject()` comes in.
 
 ```ts
+//Create the base schema without your files
 const baseSchema = z.object({
-  someOtherField: zfd.text(),
+  description: zfd.text(),
```

Comment:
Personally, I like `someOtherField` because I think it's more obvious to the reader that we're not prescribing a particular field name.